### PR TITLE
feat: Adding calls to custom style hooks derived from context in react-textarea components

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Updated components to call custom style hook

## Issues
- Updates #27058

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                          |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                      Change |
| :--------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>           |  `26.336 kB`<br />`8.585 kB` |  `29.011 kB`<br />`9.076 kB` | `2.675 kB` <br />`491 B`  |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/TextareaField.fixture.js'>TextareaField</abbr> | `36.537 kB`<br />`11.504 kB` | `39.295 kB`<br />`12.002 kB` | `2.758 kB` <br />`498 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `67.061 kB`<br />`18.105 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `205.644 kB`<br />`57.056 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `37.891 kB`<br />`12.039 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/efb2274e37c0237f3a10a4630b2696a9880f6398'>efb2274e37c0237f3a10a4630b2696a9880f6398</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Avatar-mount_base.html">1286</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Avatar-mount.html">1305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Button-mount_base.html">927</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Button-mount.html">926</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Field-mount_base.html">1957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Field-mount.html">1929</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProvider-mount_base.html">1575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProvider-mount.html">1541</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-mount_base.html">613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-mount.html">593</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">553</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">567</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/InfoButton-mount_base.html">551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/InfoButton-mount.html">518</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/MakeStyles-mount_base.html">1966</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/MakeStyles-mount.html">1943</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Persona-mount_base.html">2884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/Persona-mount.html">2832</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/SpinButton-mount_base.html">2305</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27079/perf-test-react-components/SpinButton-mount.html">2274</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: change/@fluentui-react-textarea-51a9db9d-5c84-435e-a418-46207db8398a.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: custom styles",
```
Comment (khmakoto) [inline change/@fluentui-react-textarea-51a9db9d-5c84-435e-a418-46207db8398a.json:3]:
```suggestion
  "comment": "feat: Adding calls to custom style hooks derived from context.",
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**